Releasing.md

```diff
@@ -91,6 +95,10 @@ For the body of the post, just copy this checklist and again replace all occurre
 <p>o Create a new release and name it with the tag name from step 1. For Aztec-iOS, follow <a href="https://github.com/wordpress-mobile/AztecEditor-iOS/blob/develop/Documentation/ReleaseProcess.md">this process</a>. For Aztec-Android, releases are created via the <a href="https://github.com/wordpress-mobile/AztecEditor-Android/releases">GitHub releases page</a> by hitting the “Draft new release” button, put the tag name to be created in the tag version field and release title field, and also add the changelog to the release description. The binary assets (.zip, tar.gz files) are attached automatically after hitting “Publish release”.</p>
 <!-- /wp:paragraph -->
 
+<!-- wp:paragraph -->
+<p>o Update Aztec version references within <code>gutenberg-mobile/RNTAztecView.podspec</code> and <code>gutenberg-mobile/gutenberg/packages/react-native-aztec/RNTAztecView.podspec</code> to the same, new <code>WordPress-Aztec-iOS</code> version.</p>
```

Comment:
I get what you're driving at, but "same" and "new" kind of feel like opposites so it's a bit confusing to have them both describing the same thing. Or perhaps there's a different wording that would work better. I might just drop the "same" from "the same, new...". wdyt?

Reply:
Agreed. The phrasing could be confusing. Yeah, I think dropping "same" makes sense. 
```suggestion
<p>o Update Aztec version references within <code>gutenberg-mobile/RNTAztecView.podspec</code> and <code>gutenberg-mobile/gutenberg/packages/react-native-aztec/RNTAztecView.podspec</code> to the new <code>WordPress-Aztec-iOS</code> version.</p>
```